system-int-test/test/cmr/system_int_test/ingest/generics_test.clj

```diff
@@ -82,32 +79,30 @@
           (is (string/starts-with? (:body result) "<!DOCTYPE html>") "The body is not correct"))))
 
     (testing "CREATE a document with a valid config set that includes grid"
-      (with-redefs [config/approved-pipeline-documents (fn [] {:grid ["0.0.1"]})]
-        (let [expected {:Info {:native-id native-id
-                               :document-name "Grid-A7-v1"
-                               :provider-id "PROV1"
-                               :schema "grid"
-                               :format "application/vnd.nasa.cmr.umm+json;version=0.0.1"
-                               :mime-type "application/grid;version=0.0.1"
-                               :revision-id 1
-                               :user-id "ECHO_SYS"}
-                        :Metadata gen-util/grid-good}
+      (with-redefs [config/approved-pipeline-documents (fn [] {:grid ["0.0.1"]})] 
+        (let [expected {:native-id native-id
+                        :deleted false
+                        :provider-id "PROV1"
+                        :format "UMM_JSON;0.0.1"
+                        :revision-id 1
+                        :user-id "ECHO_SYS"
+                        :concept-type "grid"
+                        :extra-fields {:document-name "Grid-A7-v1"
+                                       :schema "grid"
+                                       :mime-type "application/vnd.nasa.cmr.umm+json;version=0.0.1"}
```

Comment:
I don't agree with this being set to umm+json, it's not UMM

Reply:
I think it will be in a few months. We tried not call it UMM so it wouldn't have to go through reviews or contract deliverables.  But in 6 months to a year time, I think we will have to because someone at a DAAC site will want to create a grid.

Reply:
If it isn't then we will need to create a different mime-type.  It isn't application/json with a version such as 0.0.1.  I can write a ticket for us to do in the future.  Then we will also have to plan for when this does become a UMM type.

Reply:
I hear we will write a ticket to actually move this value to the configuration allowing each type to define the mime type to be used.

Reply:
https://bugs.earthdata.nasa.gov/browse/CMR-8614